src/legend.js

```diff
@@ -116,11 +116,12 @@ angular
           $timeout(function() {
             popoverSrv.show({
               element: el[0],
-              position: "bottom center",
+              position: 'right center',
               template:
-                '<series-color-picker series="series" onToggleAxis="toggleAxis" onColorChange="colorSelected">' +
-                "</series-color-picker>",
+                '<series-color-picker-popover series="series" onToggleAxis="toggleAxis" onColorChange="colorSelected">' +
+                "</series-color-picker-popover>",
               openOn: "hover",
+              classNames: 'drop-popover drop-popover--transparent',
```

Comment:
`dropdown-popover--transparent` will not influence current color picker. When we release the new color pickers on master it will make sure that Drop.js does not apply it's styles underneath the new popover.